packages/components/src/drop-zone/types.ts

```diff
@@ -5,6 +5,10 @@ export type DropZoneProps = {
 	 * A CSS `class` to give to the wrapper element.
 	 */
 	className?: string;
+	/**
+	 * An icon to be shown within the drop zone area.
+	 */
```

Comment:
```suggestion
	/**
	 * An icon to be shown within the drop zone area.
	 *
	 * @default "upload"
	 */
```

I'm not sure if this is the right way to express a JSX element as a default value, but I followed the format of the `DropdownMenu` component:

https://github.com/WordPress/gutenberg/blob/f1ebacf44ace3ac0a5807070b502f74b258a4a29/packages/components/src/dropdown-menu/types.ts#L65-L71

Reply:
Thanks, I was checking other examples, but didn't see this one.

P.S. It seems that `icon` prop types are defined in different ways. I initially tried using `IconProps`, but I was getting a type error. It might be a good idea to standardize this in the future.

cc @WordPress/gutenberg-components